### ISSUE TITLE
Error after installation

### ISSUE THREAD

**Issue by nanomoffet** [angular/angular-cli#1045]
> Please provide us with the following information:
> ---------------------------------------------------------------
1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)

Mac OS X Yosemite

2. Versions. Please run `ng --version`. If there's nothing outputted, please
   run in a Terminal: 
        node --version
   And paste the result here.

node --version
v5.11.1

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.

Fresh install of Node via NVM. Ran npm install -g angular-cli, completes without issue (just lodash warnings, no errors).

Unable to run ng --version, ng --help, or any other command without generating the below error.

4. The log given by the failure. Normally this include a stack trace and some
   more information.

```
/Users/josephpowers/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/node_modules/ember-cli/lib/models/project.js:669
      throw reason;
      ^

SyntaxError: /Users/josephpowers/package.json: Unexpected end of input
    at Object.parse (native)
    at Object.Module._extensions..json (module.js:430:27)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Function.Project.getProjectRoot (/Users/josephpowers/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/node_modules/ember-cli/lib/models/project.js:655:15)
    at module.exports (/Users/josephpowers/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/node_modules/ember-cli/lib/cli/index.js:51:22)
    at module.exports (/Users/josephpowers/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/lib/cli/index.js:102:10)
    at /Users/josephpowers/.nvm/versions/node/v5.11.1/lib/node_modules/angular-cli/bin/ng:31:5
```

5. Mention any other details that might be useful.
> ---------------------------------------------------------------
> > Thanks! We'll be in touch soon.

**Comment by nanomoffet** [comment]
Somehow an empty package.json was initialized in the home directory which was conflicting with it. Once I deleted that file, it started working fine.

**Comment by bmcswee** [comment]
I had the same issue today on Linux (immediately after a new global install). The empty package.json wasn't in my home dir though, it was in the parent of the dir I was in when I issued the install command.

I also solved the problem by removing this file.

**Comment by edm00se** [comment]
I had the same, with a `package.json` being created in my home dir, on Mac OS X El Capitan (10.11.5)  (#1133).

**Comment by nossareh** [comment]
I had a similar issue on Windows 10 using npm 3.10.10...
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\@angular\cli\node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"}

I did 2 things that were supported in the discussion: firstly I created the package.json and then I added the "optionalDependencies": {"fsevents: "*" }. Both of these were suggested by posts in this discussion but I can't get back to the top portion of the blog (??). So I can't get the names of the contributors. 

Now it seems to work without down grading.

Thanks
